docs/source/conf.py

```diff
@@ -21,6 +21,21 @@
 sys.path.insert(0, os.path.abspath("../../"))
 
 
+def copy_notebooks_into_docs_folder(app):
+    # .ipynb files must be inside the docs/ folder for Jupyter to be able to convert them
+    currpath = os.path.dirname(os.path.realpath(__file__))
+    source = os.path.join(currpath, "../../examples/notebooks")
```

Comment:
Can we use `Pathlib` `parent` rather than "../../" which is not portable?

Reply:
Updated